socialmediapolicy.md

```diff
@@ -0,0 +1,155 @@
+
+* Should Not - There may exist valid reasons in particular circumstances when the particular behavior is acceptable or even useful, but the full implications should be understood and the case carefully weighed before implementing.
+
+* May - Optional.
+
+## Preface
+Social media use is increasing worldwide, particularly in the younger generations. We believe it is our duty as a school to harness the opportunity we have to lead from the front and educate our children on best practice and how to stay safe online.
+
+We foster the mentality that everything should be open, transparent and public. Despite many social networks having the option to become 'private' channels, we’re fully aware that nothing online is truly private and we strive to educate our children on this viewpoint.
+
+Our school channels are all public channels. We don’t believe in closed or private groups, as this fosters the mentality of it being okay to have private conversations online, which goes against what we teach our children with regard to online safety and talking to strangers.
+
+## Our Aims
+We have three core aims for using social media within school and every action we take should be to support one or more of these aims.
+They are:
+* To increase engagement between the school and parents/carers.
+* To lead from the front and educate our children on best practice when using social media.
+* To share and promote our school in a positive light to prospective parents and the wider community.
+
+## Social Networks We Use In School
+There are many different social networks, each with a different mission, potential audience and uses. It is unrealistic to assume that we should use every single social network as a school – it would be too time consuming and difficult to manage – but we have considered each platform and actively use those that are most suitable for us and our circumstances.
+
+We believe in having a presence on all social networks regardless of whether we actively use them, since this discourages others from creating accounts which mimic or falsely represent us and our school.
+
+The school will have a presence on the following social networks: __Add Here__. Where possible, the account will be listed as a business or company under the school’s name, rather than as a personal or individual. It is important for us, as a school, to maintain a presence on these platforms to discourage others from creating a derogatory account in our school's name.
+
+The school will actively post on the following social networks: __Add Here__
+
+### Our Account Names
+* Facebook: __Add Here__
+* Twitter: __Add Here__
+* Instagram: __Add Here__
+* LinkedIn: __Add Here__
+* Snapchat: __Add Here__
+* Pinterest: __Add Here__
+* Youtube: __Add Here__
+
+## Usage and Best Practice
+
+### Taking Photographs and Videos
+
```

Comment:
Just a line...

"To be applied when capturing any content with a camera or microphone on any device."